### ISSUE TITLE
Feature Request: Simple Text Field that shows to everyone exactly like Occupation does

### ISSUE THREAD

**Issue by ViolentMinds** [SnailyCAD/snaily-cadv4#1022]
### Describe the feature

![image](https://user-images.githubusercontent.com/25211271/183266937-db386355-eb70-48db-a38c-1d8d7a44e608.png)

Could we add a text field that automatically shows to everyone just like occupation does? So its not connected to permissions. Just a default field. 

I would like my users to be able to put in their Discord nickname or whatever other name they go by like their FiveM nickname or Tiktok nickname,  YouTube name, real name lol. Whatever.

It does not have to be linked to the people who login with Discord or Steam. This is separate. Just a simple text field they can type anything into.

Basically, just add another Occupation line below the current one and call it something like "`Additional Info:`" or "`Other:`"for example. Which people can simply customize the wording of if they edit their Locales.
